dateparser_data/supplementary_language_data/date_translation_data/hi.yaml

```diff
@@ -10,6 +10,8 @@ november:
     - नवम्बर
 december:
     - दिसम्बर
+decade:
+    - दशक
```

Comment:
We can add the plural for this as well, e.g. दशकों

Reply:
@gutsytechster Thanks For Suggestions I'll be Implementing Them ASAP ....

Reply:
@gutsytechster Done this change, Please Look into it 😊.